examples/voting/src/lib.rs

```diff
@@ -210,11 +210,17 @@ fn vote<S: HasStateApi>(
     // Check that vote is in range
     ensure!(new_vote_index < host.state().num_options, VotingError::InvalidVoteIndex);
 
+    if let Some(old_vote_index) = host.state().ballots.get(&acc) {
+        let old_vote_index = *old_vote_index;
+        // Update the tally for the `old_vote_index` by reducing one vote.
+        *host.state_mut().tally.entry(old_vote_index).or_insert(0) -= 1;
```

Comment:
```suggestion
        *host.state_mut().tally.entry(old_vote_index).or_insert(1) -= 1;
```
isn't this the more correct option? I guess the old_vote_index should always exist, so the `or_insert` won't matter?